packages/core/test/issue-277.test.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var plugin = require("../plugins/scoping.js"),
+
+    processor = require("postcss")([ plugin ]);
+
+function process(src, options) {
+    return processor.process(
+        src,
+        Object.assign(Object.create(null), {
+            from  : "packages/core/test/specimens/a.css",
+            namer : (file, selector) => `a_${selector}`
+        },
+        options || {})
+    );
+}
+
+describe("/plugins", function() {
+    describe("/scoping.js", function() {
+        describe(":global issue()", function() {
+            it("shouldn't transform global selectors", function() {
```

Comment:
The test name should describe what it's checking, this doesn't seem right. Something like

```js
it("should transform multiple :global()s"
```

seems more accurate.